roles/tas_single_node/templates/scripts/backup.sh.j2

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -eu
+
+umask 077
+
+# Config Vars
+BACKUP_DIR="{{ tas_single_node_backup_restore.backup.directory }}"
+CONFIG_DIR="{{ tas_single_node_config_dir }}"
+PASSPHRASE="{{ tas_single_node_backup_restore.backup.passphrase }}"
+
+SERVICES=(
+{% if tas_single_node_trillian.database_deploy %}
+  "trillian-mysql.service"
+{% endif %}
+  "rekor.service"
+  "redis.service"
```

Comment:
Same as for the Redis volume - let's not include this if we're using external redis.